src/test/unitTests/authentication.test.js

```diff
@@ -0,0 +1,56 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
```

Comment:
We are not making any API call here. so please remove this import and where else it was used in this file. It is not necessary.